src/librbd/api/Trash.cc

```diff
@@ -25,8 +25,6 @@
 #include <json_spirit/json_spirit.h>
 #include "librbd/journal/DisabledPolicy.h"
 #include "librbd/image/ListWatchersRequest.h"
-#include <experimental/map>
-
```

Comment:
nit: why is the empty line removed? It seems we always have an empty line separator after includes.

Reply:
@trociny , thank you, Mykola! this empty line was added by accident. i just put it back in the latest revision.